### DISCUSSION TITLE
Is player in LSC?

### DISCUSSION THREAD

**Discussion by Vlhss** [YimMenu/YimMenu#1493] [Q&A]
Is it possible to check if a player is in LSC customizing vehicles?

**Comment by Tayipc38** [comment]
Probably by checking their interiors. Don't know if there is list of interior hashes but you could check what yours is while in a lsc.

**Reply by Vlhss** [reply]
nah unfortunately nothing useful from those interior globals. I basically have a check about exterior godmode and it gets triggered everytime a player is in LSC because for some reason it is considered exterior so yeah

**Reply by Tayipc38** [reply]
Have you looked at GlobalPlayerBD? Things such as `ePlayerStateFlags::kModshopActive` or `eSimpleInteriorIndex` or `GlobalPlayerBDEntry.ShopActive` look interesting to me.

**Reply by Tayipc38** [reply]
`scr_globals::globalplayer_bd.as<GlobalPlayerBD*>()->Entries[g_player_service->get_selected()->id()].PlayerStateFlags == ePlayerStateFlags::kModshopActive;`

Logic would dictate this works, but you never know.

**Reply by Vlhss** [reply]
yeah I have also tried this and it doesnt work, also I remember that playerflags are srcbitset so you have to do .value and then convert the kmodshopactive to int in order to do this correctly

**Reply by Tayipc38** [reply]
Then it must be local only. What about the other 2?

**Reply by Vlhss** [reply]
tried it locally as well, nothing came up. As for the GlobalPlayerBDEntry I think that is also local only and eSimpleInteriorIndex the index returns always 0 or -1 no matter the circumstance

**Reply by Tayipc38** [reply]
@maybegreat48 could you maybe shed light on this?
